addons/mail/static/tests/legacy/helpers/mock_server/controllers/discuss.js

```diff
@@ -545,10 +545,15 @@ patch(MockServer.prototype, {
             });
             notifications.push([
                 channel,
-                "discuss.channel/rtc_sessions_update",
+                "mail.record/insert",
                 {
-                    id: Number(channelId), // JS object keys are strings, but the type from the server is number
-                    rtcSessions: [["DELETE", notificationRtcSessions]],
+                    Thread: [
+                        {
+                            id: Number(channelId), // JS object keys are strings, but the type from the server is number
```

Comment:
Minor: there's a `parseInt(channelId)` above, could type-cast and reuse same (number) `channelId`.

This is mock server so hopefully we won't care for long...